schema/schema.go

```diff
@@ -0,0 +1,578 @@
+//The following schema package has been edited to correspond with the schematic syntax used by airbyte.
```

Comment:
Should this be copy paste or should we just import this accordingly? Was there a reason you chose to copy this code inline instead of the import?

Reply:
I would love to import it but not sure we can, the reason for the edits are that (as far as I can see) the standard schema definition only allows a single type while the airbyte implementation allows several in form of an array.

I changed it to define the types as arrays with a default null as a second type. There might be a better way of doing this, maybe by checking flags for allow null, but this would require more work I think.

Reply:
I guess we could also fork the repo and import that if you think it is better?

Reply:
Gotcha -- I think for now leaving it in the code is fine. Forking it and then having a distinct divergence which wont get merged into the mainline feels odd I think. Im good as is ✅